src/models/event-timeline-set.js

```diff
@@ -655,4 +655,6 @@ module.exports = EventTimelineSet;
  * @event module:client~MatrixClient#"Room.timelineReset"
  * @param {Room} room The room whose live timeline was reset, if any
  * @param {EventTimelineSet} timelineSet timelineSet room whose live timeline was reset
+ * @param {boolean} resetAll The return value for canResetEntireTimeline() for this room.
```

Comment:
this feels like a bit of a backwards description to me. At this point, as an app, I am interested in what the js-sdk did (ie, reset all the timelines - again, needs a better description), rather than why it did it.